lib/jekyll/converters/markdown.rb

```diff
@@ -56,21 +65,35 @@ def output_ext(ext)
       end
 
       def convert(content)
-        setup
-        @parser.convert(content)
+      setup
```

Comment:
Let's leave this indented 2 spaces to ensure it's known that it's supposed to be a part of `convert()`.